src/display/api.js

```diff
@@ -1389,6 +1391,7 @@ var PDFWorker = (function PDFWorkerClosure() {
             }
             var supportTypedArray = data && data.supportTypedArray;
             if (supportTypedArray) {
+              this.postMessageTransfers = data.supportTransfers;
```

Comment:
can you move it below in the `if (!data.supportTransfers) {` block as `this.postMessageTransfers = false;`